tests/cli/utils.js

```diff
@@ -48,7 +48,11 @@ exports.callVersionist = (opts) => {
   const cliPath = path.join(__dirname, '..', '..', 'bin', 'cli.js');
 
   const configString = _.reduce(opts, function(result, value, key) {
-    result += ` --${key}=${value}`;
+    if (key === 'cmd' || key === 'command') {
+      result = ` ${value} ${result}`;
```

Comment:
Would this line be simpler as
```
result += ` ${value}`;
```
?
(also makes it more similar to the 'else' part).

Reply:
The order would be flipped in that case, we want the command to appear first and the options after that.

Reply:
Oh, does it make a difference? (I'm not that familiar with yargs, but there's nothing in your updated README that says "commands" need to be specified before "options" ?)

Reply:
I tested some options and it seems to not matter. I am not sure if this will always hold in the future though (if we extend versionist in the future with more complex commands which take extra options).
I clarified in the readme which options are accepted for the `get`command

Reply:
Just to clarify - I was wondering if it might be worth updating the Usage string in the README to something like:
```
Usage: versionist [COMMAND] [OPTIONS]
```
? :man_shrugging: